src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -6,8 +6,41 @@ import { stateRequiredCountries } from '../../common/schemaform/definitions/addr
 import { transformForSubmit } from '../../common/schemaform/helpers';
 
 
+const siblings = ['treatments', 'privateRecordReleases', 'privateRecords', 'additionalDocuments'];
+
+export function flatten(data) {
+  const formData = Object.assign({}, data);
+  formData.disabilities.forEach((disability, idx) => {
+    siblings.forEach(sibling => {
+      if (disability[sibling]) {
+        formData[sibling] = [];
```

Comment:
For now, initializing them all to arrays is fine because they all _are_ arrays, but I don't know if that will always be true.
Maybe do something like `formData[sibling] = Array.isArray(disability[sibling]) ? [] : {}` or add a comment explaining that every sibling is an array.

Reply:
Sounds good, I've added a comment since this seems like it's specifically oriented towards managing arrays for now, if that changes in the future, the recommended change makes sense.